ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/commands/ApplyMigrationCommand.java

```diff
@@ -319,11 +320,38 @@ private void executeCommands(
       final Clock clock,
       final String previous
   ) {
+    final Map<String, Object> properties = new HashMap<>();
     final List<SqlCommand> commands = CommandParser.parse(migrationFileContent);
     for (final SqlCommand command : commands) {
       try {
-        executeCommand(command, ksqlClient, restClient);
```

Comment:
I think the code was more readable with this helper method. We can pass in `properties` to the helper method and let it manipulate the properties from within the method. If your concern was about properties changing within the helper, I don't think it's a huge deal since it's a private helper but we can also add a javadoc on the helper to make that explicit so future readers won't be confused.